modules/linear_boltzmann_solvers/discrete_ordinates_solver/lbs_discrete_ordinates_solver.cc

```diff
@@ -1192,15 +1189,15 @@ DiscreteOrdinatesSolver::InitFluxDataStructures(LBSGroupset& groupset)
             angle_indices.size(),
             dynamic_cast<const AAH_FLUDSCommonData&>(fluds_common_data));
 
-          auto angleSet = std::make_shared<TAAH_AngleSet>(angle_set_id++,
-                                                          gs_ss_size,
-                                                          gs_ss,
-                                                          *sweep_ordering,
-                                                          fluds,
-                                                          angle_indices,
-                                                          sweep_boundaries_,
-                                                          options_.sweep_eager_limit,
-                                                          *grid_local_comm_set_);
+          auto angleSet = std::make_shared<AAH_AngleSet>(angle_set_id++,
```

Comment:
`angleSet` -> `angle_set`

Reply:
Fixed. I'll have a future PR with a refactoring of the DiscreteOrdinatesSolver class as well.